procrastinate/jobs.py

```diff
@@ -49,8 +49,8 @@ class Job:
         Lock object in database that prevents other jobs with the same lock from running
         concurrently.
     queueing_lock:
-        Queueing lock object in database that prevents having several jobs with the same
-        queueing lock and in the "todo" state.
+        Lock object in database that prevents having several jobs waiting in the queue
```

Comment:
Lock is not exactly an "object". Maybe "lock name" ?

Reply:
I've used the same wording as for the `lock` argument, but I can add a commit to change the wording of both. I'll do that.

Reply:
Done. I've used the same docstrings as in the Task class, which brings extra consistency in our docs :-)